lighthouse-core/audits/dobetterweb/inspector-issues.js

```diff
@@ -29,10 +29,32 @@ const UIStrings = {
   /* eslint-enable max-len */
   /** Table column header for the types of problems observed in a website, like security or network errors. */
   columnIssueType: 'Issue type',
-  /** The type of an Issue in Chrome DevTools when a resource is blocked due to the website's cross-origin policy. */
+  /** Issue when a resource is blocked due to the website's cross-origin policy. */
   issueTypeBlockedByResponse: 'Blocked by cross-origin policy',
-  /** The type of an Issue in Chrome DevTools when a site has large ads that use up a lot of the browser's resources. */
+  /** Issue when a site has large ads that use up a lot of the browser's resources. */
   issueTypeHeavyAds: 'Heavy resource usage by ads',
+  /** Issue around "Attribution Reporting API" usage. */
+  issueTypeAttributionReporting: 'Attribution reporting',
+  /** Issue when a client hint is incorrectly used. */
+  issueTypeClientHint: 'Client hint incorrectly used',
```

Comment:
Let's use the full string `User-Agent Client Hints` - including backticks, which will tell avoid that term from being translated

so 
```
`User-Agent Client Hints` used incorrectly
```